### PR TITLE
Task/DES-2148 - Fix bugs for custom entity inputs

### PR DESCRIPTION
**Pull request by Strmiska**
## Overview: ##
This should resolve several bugs related to custom inputs on project entities...

1. User supplied inputs for project entities (experiments, simulations and hybrid simulations) should render instead of "Other" in the publication preview.
2. When switching from a custom input to a preset, the custom value should be cleared to prevent issues and it should display a preset.
3. Removed redundant object pruning before posting changes to the backend
4. Revert request queue for creating multiple experiments at once (this never happens and no one will ever need to do this)
5. Refactor some ambiguous code

## PR Status: ##

* [X] Ready.
* [ ] Work in Progress.
* [ ] Hold.

## Related Jira tickets: ##

* [DES-2148](https://jira.tacc.utexas.edu/browse/DES-2148)

## Testing Steps: ##
1. Work in an Experimental, Simulation, or Hybrid Simulation project
2. Create an Experiment, Simulation, Hybrid Simulation with custom fields. You should be able to save and edit these fields without issues (some experimental inputs cannot be changed after being created).
3. Check the publication preview area to ensure the custom values appear in the entity's metadata. They should display the custom input and not "Other"

### CODE REVIEW

**Review by SilversunKSauri — APPROVED** [review]
Tested locally, everything looks good to me

File: designsafe/static/scripts/projects/components/manage-experiments/manage-experiments.component.js [hunk 1]
```diff
@@ -333,39 +337,26 @@ class ManageExperimentsCtrl {
...
-                    name: 'designsafe.project.experiment',
-                    entity: exp,
-                },
-            }).then((res) => {
-                this.data.project.addEntity(res);
-                this.data.experiments = this.data.project.experiment_set;
-                this.data.users.forEach((user) => {
-                    user.authorship = false;
-                });
+        var experiment = this.form.addExperiments[0];
```
Comment (owaisj) [inline designsafe/static/scripts/projects/components/manage-experiments/manage-experiments.component.js:343]: Should this be declared as `let`?
Line 95 in `designsafe/static/scripts/projects/components/manage-simulations/manage-simulations.component.js` also uses `var`.

**Review by owaisj — APPROVED** [review]
LGTM
I just had one [question](https://github.com/DesignSafe-CI/portal/pull/936/files#r788006161) in the experiments and simulation component files.

File: designsafe/static/scripts/projects/components/manage-experiments/manage-experiments.component.js (hunk 1 above)
Comment (Strmiska) [reply designsafe/static/scripts/projects/components/manage-experiments/manage-experiments.component.js:343]: Oh yep, I'll swap that out

Comment (Strmiska) [reply designsafe/static/scripts/projects/components/manage-experiments/manage-experiments.component.js]: Should be sorted out now among other things.

**Pull request merged**